### DISCUSSION TITLE
23.03.22 Story 진행상황 (채하은)

### DISCUSSION THREAD

**Discussion by chaehaeun** [withPet-team/withPet-client#104] [Report]
# 🗒️ Description
늦게라도 올립니다
story 페이지가 아직 갈 길이 멀구만요...

# ✅ Todo 및 진행 상황
### 진행 상황
- [x] 카드 컴포넌트 마크업 및 스타일링
- [x] diary 작성시 id 값 추가되게 변경
- [x] diaryInfo 정보 받아와 스토리 동적 렌더링
- [x] diaryInfo의 uid를 가지고와서 petInfo 정보에서 펫 프로필 이미지 띄우기
![image](https://user-images.githubusercontent.com/103991310/227111550-108eb907-8058-47eb-91bf-20045c35bf9f.png)

### Todo
이번 주에(must!!) 해야 될 todo
- [ ] 날짜를 다이어리에 적힌 날짜(not 생성 날짜)로 수정하기
- [ ] 본문이 짧은 경우엔 더보기 버튼을 숨겨놓기
- [ ] 내 글에만 삭제 수정 버튼이 보이게 변경
- [ ] 다이어리 삭제 수정 기능
- [ ] 댓글 컴포넌트 마크업(간략한 디자인도 동시에..)/스타일링
- [ ] 댓글 작성 기능
- [ ] 댓글 누르면 인스타처럼 페이지 이동? (useParams 알아볼 것)
- [ ] 댓글 수정 삭제 기능(내 댓글만)
- [ ] 좋아요 기능
- [ ] 공유 기능? (가능 여부 확인부터)

흐린 눈한 todo
- [ ] 아이디/비번 찾기

# ETC
오늘 해결 할(하고 싶은) todo
- [ ] Story 카드 이미지 스와이퍼
- [ ] 날짜를 다이어리 작성시 설정한 날짜(not 글 작성 날짜)로 수정하기
- [ ] 본문이 짧은 경우엔 더보기 버튼을 숨겨놓기
- [ ] 내 작성 글에만 삭제 수정 버튼이 보이게 변경

**Comment by qorbaxk** [comment]
하은넴. 카드 이미지가 배열이라 여러장일경우 스와이프로 보이게끔 만들어야됩니당

**Reply by chaehaeun** [reply]
🤦‍♀️ Ok....
